joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 60

panda_1_joint_trajectory_controller:
  type: effort_controllers/JointTrajectoryController
  joints:
    - panda_1_joint1
    - panda_1_joint2
    - panda_1_joint3
    - panda_1_joint4
    - panda_1_joint5
    - panda_1_joint6
    - panda_1_joint7
  gains:
    panda_1_joint1: {p: 2000, i: 100, d: 50}
    panda_1_joint2: {p: 2000, i: 100, d: 50}
    panda_1_joint3: {p: 2000, i: 100, d: 50}
    panda_1_joint4: {p: 2000, i: 100, d: 50}
    panda_1_joint5: {p: 2000, i: 100, d: 50}
    panda_1_joint6: {p: 2000, i: 100, d: 50}
    panda_1_joint7: {p: 2000, i: 100, d: 50}
  constraints:
    goal_time: 0.5
    panda_1_joint1: { goal: 0.05 }
    panda_1_joint2: { goal: 0.05 }
    panda_1_joint3: { goal: 0.05 }
    panda_1_joint4: { goal: 0.05 }
    panda_1_joint5: { goal: 0.05 }
    panda_1_joint6: { goal: 0.05 }
    panda_1_joint7: { goal: 0.05 }

panda_2_joint_trajectory_controller:
  type: effort_controllers/JointTrajectoryController
  joints:
    - panda_2_joint1
    - panda_2_joint2
    - panda_2_joint3
    - panda_2_joint4
    - panda_2_joint5
    - panda_2_joint6
    - panda_2_joint7
  gains:
    panda_2_joint1: {p: 2000, i: 100, d: 50}
    panda_2_joint2: {p: 2000, i: 100, d: 50}
    panda_2_joint3: {p: 2000, i: 100, d: 50}
    panda_2_joint4: {p: 2000, i: 100, d: 50}
    panda_2_joint5: {p: 2000, i: 100, d: 50}
    panda_2_joint6: {p: 2000, i: 100, d: 50}
    panda_2_joint7: {p: 2000, i: 100, d: 50}
  constraints:
    goal_time: 0.5
    panda_2_joint1: { goal: 0.05 }
    panda_2_joint2: { goal: 0.05 }
    panda_2_joint3: { goal: 0.05 }
    panda_2_joint4: { goal: 0.05 }
    panda_2_joint5: { goal: 0.05 }
    panda_2_joint6: { goal: 0.05 }
    panda_2_joint7: { goal: 0.05 }

panda_1_gripper_controller:
  type: effort_controllers/GripperActionController
  joint: panda_1_finger_joint1
  gains:
    panda_1_finger_joint1: { p: 20000, i: 100, d: 50 }

panda_2_gripper_controller:
  type: effort_controllers/GripperActionController
  joint: panda_2_finger_joint1
  gains:
    panda_2_finger_joint1: { p: 20000, i: 100, d: 50 }